ADOPTERS.md

```diff
@@ -7,6 +7,7 @@ This page contains a list of organizations who are using KEDA's HTTP Add-on in p
 | Organization | Status | More Information (Blog post, etc.) |
 | ------------ | ---------| ---------------|
 |N/A|![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat)| N/A|
+|PropulsionAI|![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat)|Backend API and Frontend Web App|

```

Comment:
Please add your company URL/logo and the message (what the company does?)

Reply:
Ah! Got it.

Reply:
| Organization | Status | More Information (Blog post, etc.) |
| ------------ | ---------| ---------------|
|[![PropulsionAI](https://propulsionhq.com/assets/images/propulsion-full-color.png)](https://propulsionhq.com)|![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat)|[PropulsionAI](](https://propulsionhq.com)) allows you to add AI to your apps, without writing code.|

Reply:
@JorTurFer please review the comment. I feel the status (development and testing) is an important measure.

Reply:
Yeah, I misunderstood it xD